app/src/main/java/sdp/moneyrun/ui/map/MapActivity.java

```diff
@@ -81,13 +81,16 @@
     private String gameId;
     private boolean addedCoins = false;
     private boolean host = false;
-    private boolean useDB;
+    private final String DATABASE_COIN = "coins";
     private MapPlayerListAdapter ldbListAdapter;
-    private int coinsToPlace;
+    public int coinsToPlace;
 
     private CircleManager circleManager;
-    private int COINS_RADIUS = 6;
-    private float circleRadius = COINS_RADIUS * 10f;
+    public double game_radius;
```

Comment:
I'm sorry, I looked around and couldnt really find any justification for these attributes being public, any particular reason that you can't put them on private and add getters and setters?